### ISSUE TITLE
Add your profile card

### ISSUE THREAD

**Issue by imKashyap** [dsc-iem/AppDev-Hacktoberfest#1]
Add your profile card in the code. Each & every step is very well documented in the README as well as in comments of actual code.
Getting assigned to the issue doesn't make a difference. You raise your PR, it gets reviewed and lastly will get merged.

**Comment by AnirbanB02** [comment]
Hi @imKashyap, can you please assign this issue to me? So that I can start working on it asap. As this issue is reserved for Hacktoberfest, I'll make a PR after 1st Oct.

**Comment by imKashyap** [comment]
Alright @AnirbanB02 .

**Comment by MoyinShabi** [comment]
Hi there, can I please contribute to this also, being a beginner Flutter Dev, without being a student of your school or DSC IEM? I’m a member of DSC UNILAG in Nigeria.

I’ve been searching for repos for beginner Flutter Devs to contribute to, for Hacktoberfest 2020...

**Comment by imKashyap** [comment]
Absolutely, @MoyinShabi. Go Ahead. 👍

**Comment by MoyinShabi** [comment]
thank you!! @imKashyap

**Comment by ommiy2j** [comment]
Hi there, can I please contribute to this @imKashyap

**Comment by rajayush213** [comment]
Hi @imKashyap ! Can you assign me this issue? So that I can start contributing ASAS

**Comment by peekaboo5149** [comment]
hi @imKashyap  can you assign me this issue

**Comment by rahulsain** [comment]
hey, @imKashyap i would like to contribute to this repo

**Comment by imKashyap** [comment]
Sure @rahulsain3000 .Get it done

**Comment by ath-vk** [comment]
Hey, @imKashyap, I want to contribute to this repo, please could you assign me this issue.

**Comment by Pranjalmishra30** [comment]
Hi , @imKashyap , Can you assign this issue to me . Really excited to start contributing :))

**Comment by diksha0399** [comment]
hi, @imKashyap , i want to contribute to this repo, so please could you assign me this issue.

**Comment by imKashyap** [comment]
@Atharva-Vijay-Khade @Pranjalmishra30 fork the repo, make contribution. Your PR will be merged with first come first serve basis.

**Comment by Pranjalmishra30** [comment]
@imKashyap Yes will make the contribution and raise the PR. Do i have to be assigned the issue to make the PR ?

**Comment by imKashyap** [comment]
@Pranjalmishra30 Yeah, i have assigned a few this issue. But that doesn't make a difference. You raise your PR, it gets reviewed and lastly will get merged.

**Comment by AgungPrabowo** [comment]
Hey, @imKashyap Can you assign it to me? I will submit a PR in October as HacktoberFest starts

**Comment by vixipop** [comment]
Hey Can you assign me too, I would like to work on this!

**Comment by vixipop** [comment]
I have already submit a PR 'Jahnavi Profile card' Please check that out

**Comment by imKashyap** [comment]
Alright @jnvee

**Comment by Nimish2000** [comment]
hey @imKashyap  Can you assign it me . I would like to work on this.

**Comment by imKashyap** [comment]
Assigned @Nimish2000.

**Comment by AftabBagwan** [comment]
hey @imKashyap Can you assign it me . I would like to work on this.
